less/index.less

```diff
@@ -54,6 +55,189 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+}
+
+
+
+
+// ________________________________________Reset_______________________________________________________________________________________________
+
+@main: black;
+@accent: white;
+@nav: grey;
+@hover: darkgrey;
+@mobile: 500px;
+@h2: 3.6rem;
+
+.center() {
+	display: flex;
+	flex-direction: column;
+	margin: 0px auto;
+}
+
+.atag() {
+	text-decoration: none;
+	font-size: 2.6rem;
+	color: @nav;
+}
+
+
+
+// ________________________________________Variables/Mixins_______________________________________________________________________________________________
+
+
+.container {
+	max-width: 1080px;
+	width: 100%;
+	.center();
+	background-color: @accent;
+
+	.navbar {
+		width: 100%;
+		background-color: @main;
+		display: flex;
+		justify-content: space-between;
+		align-items: baseline;
+		border-bottom: 15px solid @accent;
+		padding: 25px;
+		
+	}
+
+	.h1 {
+		color: @accent;
+		font-size: 5.6rem;
+		font-family: 'Permanent Marker', cursive;
+	}
+
+
+	nav {
+		display: flex;
```

Comment:
I would add a wrap here. At certain widths before the mobile-breakpoint, some of the nav items are cut off.